### PR TITLE
two columns are displayed on a screen 576px

### PR DESCRIPTION
**Pull request by HryshkoAnton**

### CODE REVIEW

File: src/app/component/eco-news/components/news-list/news-list.component.scss [hunk 1]
```diff
@@ -92,9 +92,16 @@ a {
...
+}
+
+@media screen and (max-width: 575px) {
   .list {
     display: flex;
     flex-direction: column;
     align-items: center;
   }
-}
+}
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list.component.scss:107]: fix it

Comment (HryshkoAnton) [reply src/app/component/eco-news/components/news-list/news-list.component.scss:107]: what exactly do you want me to fix?

**Comment by mplesha** [comment]
conflicts

**Comment by KimAT25** [comment]
It`s looks like you and Sofia fixing the same issue! Please check my comment in PR #440 about bugfix #1033.

I described it there.

**Comment by mplesha** [comment]
conflicts

**Pull request closed without merging**